Makefile

```diff
@@ -16,7 +16,7 @@ SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo
 COMMIT_NO ?= $(shell git rev-parse HEAD 2> /dev/null || true)
 GIT_COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 
-LDFLAGS ?= -X main.gitCommit=$(GIT_COMMIT)
+GOLDFLAGS ?= -X main.gitCommit=$(GIT_COMMIT)
```

Comment:
I think `GO_LDFLAGS` would be a better name, otherwise my brain reads this as gold flags.

Reply:
Sure, I don't mind :)